market/utils/dmn-decision-table/README.md

```diff
@@ -0,0 +1,9 @@
+# Decision Table
+
+Enables you to apply DMN decision tables on your data without scripting effort.
```

Comment:
I think we had good screenshots in the old repo: they are quite expressive in showing the best supported use-case of this table. Can't we add these screenshots here gain? Or will you create new ones?

Reply:
I'll do that. @andreasbalsiger  is providing the initial README.md. And I will later add the demo section with this powerful screenshots.